src/components/Composer.vue

```diff
@@ -1008,11 +1002,9 @@ export default {
 			return `${alias.name} <${alias.emailAddress}>`
 		},
 		async discardDraft() {
-			this.state = STATES.DISCARDING
+			console.debug('discaaaaard')
 			const id = await this.draftsPromise
-			await this.$store.dispatch('deleteMessage', { id })
-			this.state = STATES.DISCARDED
-			this.$emit('close')
+			this.$emit('discardDraft', id)
```

Comment:
```suggestion
			this.$emit('discard-draft', id)
```

Reply:
actually, i used to have it like that and the button was not interacting, then Christoph sent me this: https://v2.vuejs.org/v2/guide/components-custom-events.html#Event-Names

and according to that we must write it @ discardDraft

Reply:
But the page you linked says exactly that:

> Unlike components and props, event names don’t provide any automatic case transformation. Instead, the name of an emitted event must exactly match the name used to listen to that event.

Reply:
ah, this yes :) sorry